chart_app/lib/src/helpers/chart.dart

```diff
@@ -17,6 +18,22 @@ double calculateYAxisWidth(List<Tick> ticks, ChartTheme theme, int pipSize) {
   }
 }
 
+/// Calculates the width of current tick
+double calculateCurrentTickWidth(
+    List<Tick> ticks, TextStyle textStyle, int pipSize) {
+  if (ticks.isEmpty) {
+    return 60;
+  } else {
+    final double width = labelWidth(
+      ticks.last.close,
+      textStyle,
+      pipSize,
+    );
+
+    return width + 4 * 2 + 3;
```

Comment:
why not just return 11?

Reply:
This width is based on calculation, it makes more sense to break it down than return meaningless value that would be hard to change as well as it wouldn't be clear what it's based on.

Reply:
in that case better to name them as constants so we know what each number stands for

Reply:
Well, it doesn't really stand for anything, therefore I didn't name it, it's just approximate symbol width (4) and the rest are just random multiplier and padding required to make it work. What I mean is it's same as just use + 11 but something which makes better visual representation and at least we can see it's based on some kind of calculation. I don't think I can come up with any names for these numbers.